Base/usr/share/man/man1/host.md

```diff
@@ -0,0 +1,26 @@
+## Name
+
+host - DNS lookup utility
+
+## Synopsis
+
+```**sh
+# host <name>
+```
+
+## Description
+
+`host` is a simple utility for performing DNS lookups. It is normally
+used to convert names to IP addresses and vice versa. When no arguments
+are given, `host` prints a short summary of its command line arguments.
```

Comment:
* "It is normally used" - can it be used for anything else?
* prints help with no arguments - that's... a technically correct (the best kind of correct!) way to say that it requires an argument, and prints its usage (and exits with 1) if you invoke it improperly. Let's *not* say it like that though